packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1329,6 +1330,28 @@ export default class NewVm extends BaseComponent {
 
   // DISKS -----------------------------------------------------------------------
 
+  _getSrsNotOnSameHost = createSelector(
+    () => this.state.state.existingDisks,
+    () => this.state.state.VDIs,
+    () => this.props.srs,
+    (existingDisks, vdis, srs) => {
+      let container
+      const predicate = srId =>
+        container !== undefined
+          ? container !== srs[srId].$container
```

Comment:
What if the SR is shared ?

Reply:
What if `srs` is `undefined` ?

Reply:
In which case it could be `undefined` ?

Reply:
If the data are not yet fetched.

Reply:
We can't have this case. `pool`s objects are fetched first in the selector (header), so all objects are fetched.